framework/origin_trials_client_test.py

```diff
@@ -88,6 +95,13 @@ def test_get_trials_list__with_api_key(
         'chromestatus_url': 'https://example.com/chromestatus',
         'start_milestone': '123',
         'end_milestone': '456',
+        'original_end_milestone': '450',
```

Comment:
Yes, I believe you're right. I originally implemented it this way since the listed trials were just being returned in a GET request using this helper, so it preemptively puts the data in JSON format. I'll create a change separately to update this.